draft-ietf-sedate-datetime-extended.md

```diff
@@ -132,23 +132,22 @@ in {{RFC3339}} that has the following properties:
 
 
 This document does not address extensions to the format where the
-semantic result no longer is a fixed timestamp that is referenced to a
-(past or future) UTC time.
+semantic result is no longer an unambiguous instant in time (past or
+future).
 For instance, it does not address:
 
 * Future time given as a local time in some specified time zone, where
   changes to the definition of that time zone (e.g., a political
-  decision to enact or rescind daylight savings time) changes the
-  actual time in UTC time.
-* Time given as a "floating time", i.e., a local time without
-  information as to which time zone this local time refers to.
+  decision to enact or rescind daylight saving time) affect the
+  instant in time corresponding with the timestamp.
```

Comment:
Again, I'd prefer to keep UTC here.

Reply:
What is "actual time in UTC time"? UTC is a time scale organizing instants according to calendar days and clock times within them, so "in UTC time" is clear enough. But "actual time" seems ill-defined to me. How about this?
```suggestion
  instant in UTC time corresponding with the timestamp.
```